examples/react/src/index.tsx

```diff
@@ -1,26 +1,27 @@
-import { EvolvClientOptions, EvolvProvider } from '@evolv/react';
-import { StrictMode } from 'react';
-import ReactDOM from 'react-dom';
-
+import React from 'react';
+import ReactDOM from 'react-dom/client';
 import App from './App';
-import { getUid } from './utils/uid';
+import { EvolvProvider, EvolvClientOptions } from "@evolv/react";
 
 
-const rootElement = document.getElementById('root');
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
 
-const options: EvolvClientOptions = JSON.parse(process.env.REACT_APP_EVOLV_CONFIG ?? '{}');
+const options: EvolvClientOptions = {
+  environment: "a925240014"
```

Comment:
Is this the environment ID for a common example project?

Reply:
I created a dedicated project in its own environment to use with these examples.